### ISSUE TITLE
Bugs with v-tabs and vue-router

### ISSUE THREAD

**Issue by sindrepm** [vuetifyjs/vuetify#473]
I am trying to use the `v-tabs` component to switch between different vue-router routes. My markup looks something like this:

```
<div>
<v-tabs>
  <v-tab-item ripple slot="activators" router exact href="/page1">Page 1</v-tab-item>
  <v-tab-item ripple slot="activators" router exact href="/page2">Page 2</v-tab-item>
  <v-tab-item ripple slot="activators" router exact href="/page3">Page 3</v-tab-item>
</v-tabs>

<router-view></router-view>
</div>
```

`page1`, `page2`, etc are child-routes which renders its own component within the `router-view` depending on which tab is selected. This works fine when switching between the tabs, but if I navigate directly to one of the child routes (e.g. `/page2`) there is a bug in the active tab logic. The `tab__item--active` class is applied to both the first tab item and the selected one, while the slider stays on the first item.

I also noticed that when clicking on a tab item, it adds the `tab__item--active` twice.

I can have a look at the code to see if I can fix it, but just wanted to check whether this is a known bug first?

**Comment by johnleider** [comment]
Please follow the <a href="https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md">guidelines</a> on how to report an issue. In particular, provide an example on <a href="https://jsfiddle.net/">www.jsfiddle.net</a> (or a similar service) that reproduces the problem. If necessary, create a repository for us to clone with a minimal reproduction. repositories of actual projects will generally not be accepted .

Thank you.

**Comment by sindrepm** [comment]
Sorry, I am not able to reproduce it in a fiddle because it depends on vue-router and accessing the url directly. I created a minimal repository that reproduce the issue here:

https://github.com/sindrepm/vuetify-tab-bug/blob/master/index.html

Just download the index.html and open the url like <path_to>/index.html#/page2 to see what I mean.

I added some styles to better illustrate the problem. In my real project I am using HTML history mode, but the issue exists regardless.
